recipe/meta.yaml

```diff
@@ -79,17 +79,14 @@ outputs:
         - attrs
         - click >=7.0, <=8.0.4
         - colorama
-        - dataclasses  # [py<37]
         - filelock
         - frozenlist
-        # We override the 1.43.0 upper-bound in setup.py here,
-        # see https://github.com/ray-project/ray/issues/28354
-        - grpcio >=1.32.0,<1.48
+        - grpcio >=1.32.0
         - jsonschema
         - msgpack-python >=1.0.0, <2.0.0
         - numpy >=1.20
         - pickle5  # [py<38]
-        - protobuf >=3.15.3, <4.0.0
+        - protobuf >=3.15.3
```

Comment:
Upstream has `"protobuf >= 3.15.3, != 3.19.5"`. Are we sure 3.19.5 will never be picked?

Reply:
It seems unlikely (c.f. is at protobuf 3.21 globally), but I don't mind keeping it.